src/ot_orchestration/dags/config/gentropy.yaml

```
@@ -1,17 +1,25 @@
 ---
 
-python_main_module: gs://genetics_etl_python_playground/initialisation/cli.py
-
 dataproc_cluster_settings:
-  cluster_metadata:
-    GENTROPY_REF: '{gentropy_version}'
-  cluster_init_script: gs://genetics_etl_python_playground/initialisation/install_dependencies_on_cluster.sh
-  autoscaling_policy: otg-etl
-  allow_efm: false
-  num_workers: 2
+
+  - cluster_name: otg-etl
+    cluster_metadata:
+      GENTROPY_REF: 'v{gentropy_version}'
+    autoscaling_policy: otg-etl
+    num_workers: 2
+    idle_delete_ttl: 3600 # in seconds
+
+  - cluster_name: otg-etl-coloc
+    cluster_metadata:
+      GENTROPY_REF: 'v{gentropy_version}'
+    autoscaling_policy: otg-efm
+    worker_machine_type: n1-highmem-32
+    num_workers: 20
+    allow_efm: true
```

Better to have a dict here, so we can reference keys.

```suggestion
dataproc_cluster_settings:
  default:
    cluster_name: otg-etl
    cluster_metadata:
      GENTROPY_REF: 'v{gentropy_version}'
    autoscaling_policy: otg-etl
    num_workers: 2
    idle_delete_ttl: 3600 # in seconds

  colocalisation_coloc:
    cluster_metadata:
      GENTROPY_REF: 'v{gentropy_version}'
    autoscaling_policy: otg-efm
    worker_machine_type: n1-highmem-32
    num_workers: 20
    allow_efm: true
```

---

I disagree, the names should be defined within the dictionary, this will allow for creating the config model for the operator later.